### ISSUE TITLE
The model I trained does not draw any predictions

### ISSUE THREAD

**Issue by chenyao96** [cfzd/Ultra-Fast-Lane-Detection#237]
The model I trained does not draw any predictions, is it because my data set is small? It has about 800 sheets

**Comment by cfzd** [comment]
@chenyao96 
Could you please report the training loss, top1, top2, and top3 accuracies?

**Comment by chenyao96** [comment]
> @chenyao96
> Could you please report the training loss, top1, top2, and top3 accuracies?

When I train with 800 images to 500 rounds:
iou=0.499, loss=0.391, net_time=0.108, top1=0.904, top2=0.906, top3=0.906

thanks

**Comment by cfzd** [comment]
@chenyao96 
Sorry for the very late response. Since the acc and iou are reasonable, I think you should get a good result. It might be the problem of visualization code. Could you share your visualization code?

**Comment by chenyao96** [comment]
> @chenyao96
> Sorry for the very late response. Since the acc and iou are reasonable, I think you should get a good result. It might be the problem of visualization code. Could you share your visualization code?

When I use the pre-trained model you gave, there is no problem. The same code is only loaded with different weights. It can occasionally draw some predictions when training to 500 rounds,Is it because the data set is too small

**Comment by cfzd** [comment]
@chenyao96 
It might be. Moreover, 500 rounds mean 500 batch iterations or 500 epochs? If it is 500 batch iterations, the training might be very insufficient.

**Comment by chenyao96** [comment]
> @chenyao96
> It might be. Moreover, 500 rounds mean 500 batch iterations or 500 epochs? If it is 500 batch iterations, the training might be very insufficient.

500 epochs, I will try enlarge the data set, thank you very much

**Comment by chenyao96** [comment]
> @chenyao96
> It might be. Moreover, 500 rounds mean 500 batch iterations or 500 epochs? If it is 500 batch iterations, the training might be very insufficient.

Hello author, my training weight is much larger than yours, mine is 1.1G, you gave 245M, and I also use resnet18 network

**Comment by cfzd** [comment]
@chenyao96 
You can check this issue: #108
